packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -116,4 +116,20 @@ class DateTimeUtils {
   }
 }
 
+// These formats are designed to be the possible props for the
```

Comment:
I am curious, should we redefine them here again or could we re-use what [TimeInput](https://github.com/cerner/terra-framework/blob/master/packages/terra-time-input/src/TimeUtil.js#L199) already provides?

Reply:
changes done for variant type of date-time-picker : 44faa6c1df1f928ccce787ef4cded5943b3c46e5


@bjankord  @ryanthemanuel
can we update the [examples](https://github.com/cerner/terra-framework/blob/d7dc43cd32aa3dcb043b41fd059749e6628f2cad/packages/terra-time-input/src/terra-dev-site/doc/example/TimeInputDefaultTwelveHour.jsx#L27) of Time input also to replace the hard code values for time-format with Time util Format like :  ```` variant 
 = {TimeUtil.FORMAT_12_HOUR} ````

Reply:
Yes, feel free to update it. But you would also need to update the Changelog in time-input for this change.

Reply:
thank you, updated time input docs and example : d239333e941e60599b1818f8447e08fd4fdc10a2